src/bridge/IrcBridge.ts

```diff
@@ -1486,26 +1486,32 @@ export class IrcBridge {
         log.info(`Ghost migration to ${newRoomId} complete`);
     }
 
-    public async connectionReap(logCb: (line: string) => void, reqServerName: string,
-                                maxIdleHours: number, reason = "User is inactive", dry = false,
-                                defaultOnline?: boolean, excludeRegex?: string, limit?: number) {
+    /**
+     * Calculate the number of idle users
+     * @param server The IRC server which we want to scope the idle check to.
+     * @param minIdleHours The minimum number of hours to be considered idle.
+     * @param defaultOnline Whether the user should be defaulted to online or offline if we hold no data for them.
+     * @param excludeRegex A regex of users to exclude from the check.
+     * @param maxIdleHours The maximum number of hours to be considered
+     *                     idle before they aren't considered part of the pool. By default, this isn't checked.
+     * @returns A ordered set of userIds by their idle time in ascending order.
+     */
+    public async calculateIdlenessPool(
+        server: IrcServer, minIdleHours: number,
+        defaultOnline?: boolean, excludeRegex?: string,
```

Comment:
```suggestion
        defaultOnline: boolean, excludeRegex?: string,
```

We're only calling it with a defined boolean – allowing undefined brings a question of "but what happens if you don't pass it in" which will be confusing since it defaults to true on the API side of things. (Same in the other methods).